{{cookiecutter.project_name}}/docker/django/Dockerfile

```diff
@@ -69,10 +70,10 @@ COPY ./docker/django/entrypoint.sh /docker-entrypoint.sh
 
 # Setting up proper permissions:
 RUN chmod +x '/docker-entrypoint.sh' \
-  && groupadd -r web && useradd -d /code -r -g web web \
-  && chown web:web -R /code \
+  && groupadd -r ${USER} --gid ${USER_GID} && useradd --uid ${USER_ID} --gid ${USER_GID} -d /code -r -g ${USER} ${USER} \
```

Comment:
We use `$USER` and `$USER_GID` notations

Reply:
I think username and group name can be same.